### PR TITLE
Add isEmptyOrWhitespaceString method

### PR DESCRIPTION
**Pull request by Benuuu**

### CODE REVIEW

**Comment by vinnyt** [comment]
:shipit:

**Comment by loganwright** [comment]
Let's rename to `isEmptyOrWhiteslace` w/o `String` suffix. 

:ship-it:

File: SwiftWisdom/Core/StandardLibrary/String/String+Empty.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//
+
+import Foundation
+
+extension String {
+    public func ip_isEmptyOrWhitespaceString() -> Bool {
+        return stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceAndNewlineCharacterSet()).isEmpty
+    }
+
+    public func ip_isNonEmpty() -> Bool {
```
Comment (paulrolfe) [inline SwiftWisdom/Core/StandardLibrary/String/String+Empty.swift]: `ip_isNonEmptyOrWhitespaceString` or take it out?

**Comment by Benuuu** [comment]
@LoganWright renamed to `ip_isEmptyOrWhitespace`
@paulrolfe removed the negation method

**Comment by paulrolfe** [comment]
:sake:

**Pull request merged**